tests/testthat/helper.R

```diff
@@ -0,0 +1,43 @@
+# Define the URLs
+urls <- c(
+  "https://lw-enram.s3-eu-west-1.amazonaws.com/be/jab/2016/09/19/23/bejab_vp_201609192315.h5",
```

Comment:
Don't use lw-enram bucket, it will be deleted. The equivalent file is:

```
https://lw-enram.s3-eu-west-1.amazonaws.com/ecog-04003/hdf5/bejab/2016/09/19/bejab_vp_201609192315.h5
```

But it might be better to use an equivalent of the vpts file from 2023

Reply:
Note: choose something from aloft - aloft.s3-eu-west-1.amazonaws.com

Reply:
Indeed, I wrote the wrong link, now corrected.